packages/framework-stats/src/Packagist/Purifier/PHPStanNettePackagesPurifier.php

```diff
@@ -34,7 +34,7 @@ public function correctLastYearDownloads(int $yearDownloads, string $packageName
 
         // packagist doesn't provide way to count this number, so it is estimation compared to another nette packages,
         // and phpstan downloads from https://packagist.org/packages/phpstan/phpstan/stats
-        $phpstanLastYearInfluence = 3_500_000;
+        $phpstanLastYearInfluence = 1_500_000;
```

Comment:
PHPStan dependency on Nette is long gone, so it's time to lower the penalty for last year PHPStan influence